test/Altinn.Profile.Tests/Profile.Integrations/UserPreferences/PartyGroupRepositoryTests.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Altinn.Profile.Core.PartyGroups;
+using Altinn.Profile.Integrations.Persistence;
+using Altinn.Profile.Integrations.Repositories;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using Xunit;
+
+namespace Altinn.Profile.Tests.Profile.Integrations.UserPreferences
+{
+    public class PartyGroupRepositoryTests : IDisposable
+    {
+        private bool _isDisposed;
+        private readonly ProfileDbContext _databaseContext;
```

Comment:
I'm wondering if it could be a good idea to rewrite the data seeding logic below to not rely on a class field for the DbContext. Use new instead. 

It's fine because we're getting a new instance of the test class for every test, but not knowing this it looks like we're reusing the DbContext for different "unit of work".